include/secp256k1_rangeproof.h

```diff
@@ -286,6 +286,27 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_rangeproof_info(
   size_t plen
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4) SECP256K1_ARG_NONNULL(5);
 
+/** Verify a rangeproof with a single-value range. Useful as a "proof of value"
+ *  of a Pedersen commitment. Such proofs can be created with `secp256k1_rangeproof_sign`
+ *  by passing an `exp` parameter of -1 and the target value as both `value` and `min_value`.
+ *  Returns 1: Proof was valid and proved the given value
```

Comment:
Is it also the case that `min_bits` must be 0 for the rangeproof, (as elements generates it) - should this be documented here?

Reply:
I just checked, and interestingly no -- you can set `min_bits` to anything and it'll just be ignored.

Reply:
Added a comment to this effect.

Reply:
You gave a ":+1:" but I think this hasn't been fixed.